src/types/wbgetentities.ts

```diff
@@ -0,0 +1,13 @@
+import type { UrlResultFormat } from './options.js'
+
+export type WbGetEntities = {
+  action: 'wbgetentities',
+  titles?: string,
+  sites?: string,
+  ids?: string,
+  format: UrlResultFormat,
+  normalize?: string,
```

Comment:
According to the [docs](https://www.wikidata.org/w/api.php?action=help&modules=wbgetentities) normalize is a boolean which is either present with any value (true) or not present (false).

This would force it to be either specified with true or omitted:

```suggestion
  normalize?: true,
```